pkg/utils/utils.go

```diff
@@ -31,6 +31,17 @@ const (
 	TiB = 1024 * GiB
 )
 
+var (
+	// ReservedKeywords are Glusterd2 reserved keywords
+	ReservedKeywords = []string{"all", "volume", "status", "list", "cluster", "gluster", "brick"}
```

Comment:
add comment for why we are reserving these keywords

Reply:
Reserved key words is self explanatory

Reply:
I wanted to see, why we are reserving these keywords, this will help the developers who will join later.
by these comments, I don't think they will get more context why these keywords are reserved.